.bazelversion

```diff
@@ -1 +1 @@
-3.1.0
\ No newline at end of file
+3.7.2
```

Comment:
Is the bazel version 3.7.2 necessary? If it is, please also mention it in README.